lld/COFF/Driver.cpp

```diff
@@ -365,13 +365,20 @@ bool LinkerDriver::isDecorated(StringRef sym) {
          (!ctx.config.mingw && sym.contains('@'));
 }
 
-// Parses .drectve section contents and returns a list of files
-// specified by /defaultlib.
-void LinkerDriver::parseDirectives(InputFile *file) {
-  StringRef s = file->getDirectives();
-  if (s.empty())
+void LinkerDriver::processDrectveSections(InputFile *file) {
+  std::vector<StringRef> directivesList = file->getDrectves();
+  if (directivesList.empty())
     return;
+  for (StringRef s : directivesList) {
```

Comment:
Can we just write `for (StringRef s : file->getDirectives())`, without all the above?